app/react/sidebar/UpgradeBEBanner/UpgradeBEBanner.tsx

```diff
@@ -22,8 +22,8 @@ export const UpgradeBEBannerWrapper = withHideOnExtension(
 );
 
 const enabledPlatforms: Array<ContainerPlatform> = [
-  'Docker Standalone',
   'Docker Swarm',
+  'Docker Standalone',
   'Kubernetes',
 ];
```

Comment:
Does the order matter? If no, revert the changes. If yes, add a code comment to explain.